pkg/controller.v1/common/job_controller.go

```diff
@@ -69,9 +69,9 @@ var (
 type GangScheduler string
 
 const (
-	GangSchedulerNone             GangScheduler = "None"
```

Comment:
oh I wasn't aware that this existed. Maybe we can add another:

```
case GangSchedulerNone, "":
  return nil
default:
  // scheduler plugins
```

Reply:
Done